src/classes/REL_Relationships_TDTM.cls

```diff
@@ -218,8 +210,7 @@ public with sharing class REL_Relationships_TDTM extends TDTM_Runnable {
         for(npe4__Relationship__c r : Relationships) {
         	
             //SWA 2009-05-13 removed a duplicate check for change to npe4__Status__c from this OR statement
-            if ((r.npe4__Status__c != oldRelationships[counter].npe4__Status__c 
-            || r.npe4__Type__c != oldRelationships[counter].npe4__Type__c 
+            if ((r.npe4__Type__c != oldRelationships[counter].npe4__Type__c 
```

Comment:
but does this mean making one former will not update the reciprocal to former?  I thought #1584 was about us messing up the gender rather than leaving it alone.